src/rendering/screenspacerenderable.cpp

```diff
@@ -170,28 +168,37 @@ namespace {
     constexpr openspace::properties::Property::PropertyInfo BorderColorInfo = {
         "BorderColor",
         "Border Color",
-        "Sets the color of the border.",
+        "The color of the border.",
         openspace::properties::Property::Visibility::NoviceUser
     };
 
     float wrap(float value, float min, float max) {
         return glm::mod(value - min, max - min) + min;
     }
 
+    // This is the base class of all `ScreenSpaceRenderable` types, which are objects
+    // that are rendered in their own coordinate system on top of the other 3D rendering.
```

Comment:
```suggestion
    // that are rendered in their own layer on top of the other 3D rendering.
```
Not sure about this.  Weighing between making it easier to understand vs completeness

Reply:
We can also refer to it as "in front of the camera"

Reply:
I applied the suggestions where you suggested this alternative phrasing, Ylva. What's your thoughts on this way of explaining it @alexanderbock ?

Also, I think that this sentence might be a bit hard to understand in genera. What do you think of this formulation instead (essentially combining your suggestions :) )? 
```suggestion
    // that are rendered in a layer in front of the camera, separate from the rest of the 3D rendering.
```

Reply:
The coordinate system is mentioned in the next paragraph, so I think it's ok to remove it from here